naas/models/projects.py

```diff
@@ -89,3 +104,9 @@ def create(params=None):
 
         Configuration({"logger": f"failure_message"})
         raise InvalidRequestError(failure_message)
+
+    def to_a(self):
+        """
+        Returns the collection serialized as an array
+        """
+        return list(self.collection)
```

Comment:
This is the wrong implementation. It's not just a list of the collection, it's a list of the decorated objects. This currently throws an error.